<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Merchandise_Sales</class>
 <widget class="QDialog" name="Merchandise_Sales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merchandise Sales</string>
  </property>
  <widget class="QPushButton" name="MerchandiseSales_BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MerchandiseSales_NumberSearch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesNov15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Nov '15</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesJan16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Jan '16</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesJun15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales June '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxApr16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesDec15">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Dec '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxDec15">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesOct15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Oct '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxFeb16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesSep15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Sept '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxJul15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxSep15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="MerchandiseSales_SearchButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumNumberBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxMar16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesMar16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Mar '16</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_NumberText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Item ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesApr16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Apr '16</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxOct15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxAug15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxJan16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesAug15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Aug '15</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesFeb16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales Feb '16</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesJul15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales July '15</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxMay16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MerchandiseSales_TitleSearch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_TitleText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxJun15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="MerchandiseSales_SalesBoxNov15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_MainText">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>20</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Merchandise Sales</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="AlbumSales_SaveButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumTitleBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AlbumSales_AlbumSalesMay16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sales May '16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AlbumSales_PrintButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>MerchandiseSales_BackButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Sales</receiver>
   <slot>BackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MerchandiseSales_SearchButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Sales</receiver>
   <slot>SearchMerchandise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AlbumSales_PrintButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Sales</receiver>
   <slot>Print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AlbumSales_SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Sales</receiver>
   <slot>SaveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackButton()</slot>
  <slot>SearchMerchandise()</slot>
  <slot>Print()</slot>
  <slot>SaveChanges()</slot>
 </slots>
</ui>
